### DISCUSSION TITLE
Roadmap for UCI

### DISCUSSION THREAD

**Discussion by Krishnaj20** [samagra-comms/community#23] [General Discussion]
@aayushkedia95 - Sunbird ED is using UCI for the whatsapp chatbot capability. Is there a roadmap for UCI, we can refer to? Wanted to know if there are any enhancements you are planning to UCI.

**Comment by aayushkedia95** [comment]
Hi @Krishnaj20, I'm marking @ChakshuGautam @abhay-samagra who will be able to provide clarity on this.
